share/spack/gitlab/cloud_pipelines/stacks/developer_tools/spack.yaml

```diff
@@ -0,0 +1,65 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=x86_64_v3
+  concretizer:
+    reuse: true
+    unify: true
+  definitions:
+    - default_specs:
+      # editors
+      - neovim~no_luajit
```

Comment:
I have a personal preference for making the variants we test in CI the defaults in Spack. Is there a strong reason we should leave `no_luajit` as default True in neovim? Or that we need to enable luajit here?

Reply:
That option is confusingly named.  When it's on, use lua, when it's off use luajit.  I think this *is* the default, but unless it's forced something else was causing lua to get built instead of luajit, and it causes neovim to behave badly.